src/transactions/TransactionTable.js

```diff
@@ -4,19 +4,29 @@ import { v4 as uuid } from 'uuid';
 import { useSelector } from 'react-redux';
 import TransactionRow from './TransactionRow';
 import TransactionForm from './TransactionForm';
-
+// import transactionFilter from './test';
 /**
  * Shows a table of transactions and optionally a form for creating a transaction
  *
  * @component
  * @example
  * <TransactionTable isCreating onStopCreating={() => alert('finished')} />
  */
-function TransactionTable({ isCreating = false, onStopCreating }) {
+
+//added filters
+function TransactionTable({ isCreating = false, onStopCreating, filters }) {
+
   // Get all the transactions to show in the table
-  const transactionList = useSelector((state) => state.transactions.entities);
 
-  // JSX
+  //original code
+  // const transactionList = useSelector((state) => state.transactions.entities);
+
+  // add a prop here for filters
+
+  //keeping useSelector and state.transaction.entities
+  const transactionList = useSelector((state) => state.entities.filter((t) => t.categoryId === filters.categoryId && t.payee === filters.payee))
+
```

Comment:
Still need to add date filtering and make sure it handles if they don't set a filter (meaning categoryId/payee/startData/endDate would be null or undefined)